Doc/README.rst

```diff
@@ -7,45 +7,49 @@ available <https://docs.python.org/dev/download.html>`_.
 
 Documentation on authoring Python documentation, including information about
 both style and markup, is available in the "`Documenting Python
-<https://docs.python.org/devguide/documenting.html>`_" chapter of the
+<https://devguide.python.org/documenting/>`_" chapter of the
 developers guide.
 
 
 Building the docs
 =================
 
-You need to have `Sphinx <http://sphinx-doc.org/>`_ installed; it is the toolset
-used to build the docs.  It is not included in this tree, but maintained
-separately and `available from PyPI <https://pypi.python.org/pypi/Sphinx>`_.
+The documentation is built with several tools which are not included in this
+tree but are maintained separately and are available from
+`PyPI <https://pypi.org/>`_.
 
+* `Sphinx <https://pypi.org/project/Sphinx/>`_
+* `blurb <https://pypi.org/project/blurb/>`_
 
-Using make
-----------
+The easiest way to install these tools is to create a virtual environment and
+install the tools into there.
 
-A Makefile has been prepared so that on Unix, provided you have installed
-Sphinx, you can just run ::
 
-   make html
-
-to build the HTML output files.
+Using make
+----------
 
-On Windows, we try to emulate the Makefile as closely as possible with a
-``make.bat`` file.
+To get started on UNIX, you can create a virtual environment with the command ::
 
-To use a Python interpreter that's not called ``python``, use the standard
-way to set Makefile variables, using e.g. ::
+  make venv
 
-   make html PYTHON=python3
+That will install all the tools necessary to build the documentation. Assuming
+the virtual environment was created in the ``env`` directory (the default;
+configurable with the VENVDIR variable), you can run the following command to
+build the HTML output files::
 
-On Windows, set the PYTHON environment variable instead.
+  make html PYTHON=./env/bin/python
```

Comment:
Alas, this is currently insufficient.  The `sphinx-build` command in the venv won't be found without other Makefile changes (it doesn't fail if you already have `sphinx-build` installed on your PATH).  Since we're going to have tweak this a bit once the `blurb` support is merged in, suggest for now just adding a shell source command prior to the `make html`, i.e. `. ./env/bin/activate` or (`. ./venv/bin/activate`).